File: 1 semestr/lab6/lab6_4.py
def element(x):
    a = [0,1]
    for i in range(2,x+1):
        a.append(a[i-2]+a[i-1])
    return a[x]
k = int(input("Введите k-тое число: "))
n = int(input("Введите n-тое число: "))
s = element(k)/element(n)
print ("Отношение k/n:", round(s,3))

File: 1 semestr/lab6/lab6_4/lab6_4.cpp
#include <iostream>
#include <ctime>
#include <cmath>
using namespace std;


float element(int x) {
    srand(time(NULL));
    int a[100] = { 0,1 };
    for (int i = 2; i < x+1; i++) {
        a[i] = a[i-2]+a[i-1];
    }
    return a[x];
}

int k, n;
float s;

int main()
{
    cin >> k;
    cin >> n;
    s = element(k) / element(n);
    cout << round(s * 1000) / 1000;
}
